docs/reference/cluster/stats.asciidoc

```diff
@@ -1522,7 +1521,11 @@ This may include requests where partial results were returned, but not requests
 
 =======
 
+
 ======
+`_esql`:::
+(object) Contains the information about the ES|QL <<esql-cross-clusters,cross-cluster queries>> usage in the cluster.
```

Comment:
nit: Product doesn't want to call this "cross-cluster query", but rather "ES|QL cross-cluster search". And then you can use the `{ccs}` macro available in the docs to ensure it is consistently used/hyphenated/etc.